assignment1/date_manipulation.rb

```diff
@@ -0,0 +1,35 @@
+require 'date'
+
+puts 'a. Print current date.'
+
+time = Time.now
+puts time
+
+puts '
+b. Print 12th Jan 2012'
+
+puts Time.new(2012, 1, 12).strftime("%d %b %Y")
```

Comment:
please use 'Date' instead of 'Time', output  "12 Jan 2012" received instead of "12th Jan 2012", can ou please check this